eeg_publisher/mock_publisher.py

```diff
@@ -13,8 +14,13 @@
 class MockEEGPublisher(Node):  # type: ignore[misc]
     def __init__(self) -> None:
         super().__init__("mock_eeg_publisher")
-        self.n_seed: int = 0
+        self.bridge = EEGBridge()
 
+        # logger parameters
+        self.throttle_duration: float = 1.0
+        self.once: bool = False
+
+        self.n_seed: int = 0
         self.queue_size: int = 10
         self.num_channels: int = 8
         self.num_samples: int = 32
```

Comment:
To avoid hard-coding init, I suggest adding a YAML file (maybe in a new path called configs) to set up parameters either to [Setting initial parameter values when launching nodes](https://docs.ros.org/en/humble/Concepts/Basic/About-Parameters.html#id8) or [Manipulating parameter values at runtime with ros2 param](https://docs.ros.org/en/humble/Concepts/Basic/About-Parameters.html#id9)[](https://docs.ros.org/en/humble/Concepts/Basic/About-Parameters.html#manipulating-parameter-values-at-runtime)

* https://docs.ros.org/en/humble/How-To-Guides/Using-ros2-param.html
* https://docs.ros.org/en/humble/How-To-Guides/Node-arguments.html#setting-parameters-from-yaml-files

Reply:
fyi. this is the log for  `ros2 param list`
```
 ros2 param list
/eeg_publisher1/mock_publisher:
  use_sim_time
/eeg_publisher1/mock_subscriber:
  use_sim_time
  ```